ioctl/newcmd/action/action_test.go

```diff
@@ -25,15 +27,15 @@ func TestSigner(t *testing.T) {
 	require := require.New(t)
 	ctrl := gomock.NewController(t)
 	client := mock_ioctlclient.NewMockClient(ctrl)
-	client.EXPECT().SelectTranslation(gomock.Any()).Return("mockTranslationString", config.English).Times(3)
+	client.EXPECT().SelectTranslation(gomock.Any()).Return("mockTranslationString", config.English).AnyTimes()
 	client.EXPECT().SetEndpointWithFlag(gomock.Any()).Do(func(_ func(*string, string, string, string)) {})
 	client.EXPECT().SetInsecureWithFlag(gomock.Any()).Do(func(_ func(*bool, string, bool, string)) {})
 
 	t.Run("returns signer's address", func(t *testing.T) {
 		client.EXPECT().AddressWithDefaultIfNotExist(gomock.Any()).Return("test", nil).AnyTimes()
 
 		cmd := NewActionCmd(client)
-		registerSignerFlag(client, cmd)
```

Comment:
does registerSignerFlag run failed?

Reply:
I replace it with `RegisterWriteCommand`, which will execute `registerSignerFlag` in the process.

Reply:
No need to replace because calling `registerSignerFlag` is the final aim for this test.